src/views/Verbose.vue

```diff
@@ -71,6 +71,41 @@
             v-model="filter"
             :placeholder="$t('verbose.filterPlaceholder')"
           >
+          <table>
+            <tr>
+              <td>
+                <div
+                  :class="{ 'exclude-result': true, 'selected': isExcluded('true') }"
+                  @click="excludeResult('true')"
+                >
+                  <span></span>
+                </div>
+              </td>
+              <td>{{ $t('verbose.exclude') }} <code :class="'true'">true</code></td>
+            </tr>
+            <tr>
+              <td>
+                <div
+                  :class="{ 'exclude-result': true, 'selected': isExcluded('false') }"
```

Comment:
This class can be written like

`:class="['exclude-result', { selected: isExcluded('false') }]"`